runtime/lua/vim/lsp/completion.lua

```diff
@@ -706,6 +853,36 @@ local function enable_completions(client_id, bufnr, opts)
         end
       end,
     })
+
+    if vim.o.completeopt:find('popup') then
+      api.nvim_create_autocmd('Completechanged', {
+        group = group,
+        buffer = bufnr,
+        callback = function(args)
+          local completed_item = api.nvim_get_vvar('event').completed_item or {}
+          if (completed_item.info or '') ~= '' then
+            local data = vim.fn.complete_info({ 'selected' })
+            update_popup_window(data.preview_winid, data.preview_bufnr)
+            return
+          end
+
+          local client = lsp.get_client_by_id(
+            vim.tbl_get(completed_item, 'user_data', 'nvim', 'lsp', 'client_id')
+          )
+
+          if not client or not client:supports_method(ms.completionItem_resolve) then
+            return
+          end
```

Comment:
```suggestion
          local clients = lsp.get_clients({
            id = vim.tbl_get(completed_item, 'user_data', 'nvim', 'lsp', 'client_id'),
            method = ms.completionItem_resolve
          )

          if #clients == 0 then
            return
          end
```
WDYT?

Reply:
better

Reply:
And `clients` became redundant

Reply:
I don't use that just inline check size of table.